sources/src/App.vue

```diff
@@ -52,12 +52,12 @@ export default {
               {
                 header: "CLEAR",
                 text:
-                  '<p> Robot Framework has a modular architecture that can be extended with bundled and self-made <a href="#libraries">test libraries</a>. </p> <p> Test data is defined in files using the syntax shown in the <a href="#examples">examples</a> below. A file containing test cases creates a test suite and placing these files into directories creates a nested structure of test suites. </p>'
+                  '<p> Robot Framework has a modular architecture that can be extended with bundled and self-made <a href="#libraries">libraries</a>. </p> <p> Data is defined in files using the syntax shown in the <a href="#examples">examples</a> below. A file containing tasks or test cases creates a suite, and placing these files into directories creates a nested structure of suites. </p>'
               },
               {
                 header: "EASY",
                 text:
-                  "<p> When test execution is started, the framework first parses the test data. It then utilizes keywords provided by the test libraries to interact with the system under test. Libraries can communicate with the system either directly or using other test tools as drivers. </p> <p> Test execution is started from the command line. As a result you get report and log in HTML format as well as an XML output. These provide extensive look into what your system does. </p>"
+                  "<p> When execution is started, the framework first parses the data. It then utilizes keywords provided by the libraries to interact with the target system. Libraries can communicate with the system either directly or using other tools as drivers. </p> <p> Task or test execution is started from the command line. As a result, you get a report and log in HTML format as well as an XML output. These provide an extensive look into what your system does. </p>"
```

Comment:
- Good additions of "a" and "an".
- Talking about "execution" instead of "test execution" is fine.
- If you refer to both, use "test or task execution", not "task or test execution".